studiopress/genesis-custom-blocks
# Merge editor feature branch into develop

#### Changes
* Now that the editor is released, this just merges the editor UI feature branch into `develop`, the default branch
* This feature branch was so that we could still do minor bugfix releases in the months while the editor was in development
* Carries out number 10 of the [Release Procedure](https://github.com/studiopress/genesis-custom-blocks/blob/15dc077cb5c422174157495e761d0f696984dd5f/CONTRIBUTING.md#release-procedure)

#### Testing instructions
* Not needed, just a sanity-check of the code (are there files that definitely shouldn't be there?)
* Every commit except the one to [resolve a merge conflict](https://github.com/studiopress/genesis-custom-blocks/pull/58/commits/1db2deb6eed0d32f9f2b148364999e21bae4f83b) was part of a PR that was already reviewed. The PRs were just to the `feature/react-ui` branch.
